### PR TITLE
add fix script service and acpi stuff

### LINKED ISSUE #67: Support fn+r refresh rate toggle
now I use a script to toggle refresh rate

https://github.com/st0nie/refreshd

and acpid event file
```conf
event=1E3391A1-2C89- 000000e8 00000000
action=sudo -u ston HOME=/home/ston/ DIPLAY=":1" XDG_RUNTIME_DIR=/run/user/1000 QT_QPA_PLATFORM=wayland /home/ston/.local/bin/refreshed toggle
```

But there is a problem with this. any fn+letter key combination triggers this toggle. Is there a way to only use fn+q?

### PR DESCRIPTION
**Pull request by MrDuartePT**
I get the service script to work and add some acpi action that need to be implemet (i talking about the fn+PrtSc, fn+novo, fn+R)
But this can be done later

### CODE REVIEW

**Comment by MrDuartePT** [comment]
Probably the build are going to fail because my branch is up to date with the latest commit

**Comment by MrDuartePT** [comment]
Besides the acpi the service is working good

**Comment by MrDuartePT** [comment]
not it all good if you like it merge it

File: extra/acpi/battery-legion-quiet.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+ADP0_connected=$(cat /sys/class/power_supply/ADP0/online)
+
+if [[ $ADP0_connected -eq 1 ]]; then
+	/bin/echo balanced > /sys/firmware/acpi/platform_profile
+else
+	/bin/echo quiet > /sys/firmware/acpi/platform_profile
+fi
+
+/bin/systemctl restart lenovo-fancurve.service #systemd restart on profile change
```
Comment (MrDuartePT) [inline extra/acpi/battery-legion-quiet.sh:11]: i go to update this

**Comment by st0nie** [comment]
https://github.com/Petingoso/legion-fan-utils-linux/issues/11

If you have a short time, it may cause this, better fix the problem, thank you

**Comment by st0nie** [comment]
图片

And, we should standardize the naming of the preset files to be consistent with the gui,

For example, the preset in the script is now called quiet-bat, but in legion_gui it is quiet-battery

File: extra/service/legion-linux.service [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Apply Fan Curve to Lenovo Legion Laptop depending of Power State and Charger Mode (LenovoLegionLinux kernel Module)
+DefaultDependencies=no
+StopWhenUnneeded=yes
+After=systemd-logind.service
+
```
Comment (st0nie) [inline extra/service/legion-linux.service:6]: add `StartLimitIntervalSec=0` here to prevent limit-hit

File: extra/service/profiles/balanced-bat.yaml
Comment (st0nie) [inline extra/service/profiles/balanced-bat.yaml:1]: rename it to balanced-battery will be better

File: extra/service/profiles/quiet-bat.yaml
Comment (st0nie) [inline extra/service/profiles/quiet-bat.yaml:1]: the same, rename it

File: extra/service/fancurve-set [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        echo "Applying Balance Mode Profile   -> battery..."     
+        GPU_TDP=140 #set to GPU 65W
+        FANCURVE_FILE=balanced-performance-ac #set the fancurve file
+        CPU_CONTROL_COMMAD=$(echo test)
+    fi
+else
+    if [ $POWER_PROFILE == quiet ]; then
+        echo "Applying Quiet Mode Profile ﴛ  -> battery..."
+        GPU_TDP=55 #set to GPU 55W
+        FANCURVE_FILE=quiet-bat #set the fancurve file
```
Comment (st0nie) [inline extra/service/fancurve-set:54]: quiet-battery

File: extra/service/fancurve-set [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        CPU_CONTROL_COMMAD=$(echo test)
+
+    elif [ $POWER_PROFILE == balanced ]; then
+        echo "Applying Balance Mode Profile   -> battery..."     
+        GPU_TDP=65 #set to GPU 65W
+        FANCURVE_FILE=balanced-bat #set the fancurve file
```
Comment (st0nie) [inline extra/service/fancurve-set:60]: balanced-battery

**Review by st0nie — CHANGES_REQUESTED** [review]

File: extra/service/legion-linux.service (hunk 2 above)
Comment (MrDuartePT) [reply extra/service/legion-linux.service:6]: Yes that a good ideia

**Comment by MrDuartePT** [comment]
> 图片
> 
> And, we should standardize the naming of the preset files to be consistent with the gui, For example, the preset in the script is now called quiet-bat, but in legion_gui it is quiet-battery
> 
> And if the preset is not found in the user's ~/.config/legion_linux/, we should help the user get the preset from /usr/share/legion_linux.

i only do that on the package i mantain like the arch and gentoo package when you are installing from the script it will be install in the proper directory but i can also make a copy to /usr/share

**Comment by MrDuartePT** [comment]
@johnfanv2 verify everthing but i think is good to go

**Comment by st0nie** [comment]
Good job, but I think there are some things that could be improved, the GPU_TDP of fancurve-set is hardcoded, we should let the user change the GPU_TDP via .env, because the file fancurve-set is no longer in /etc, but in /usr/bin, in this case it should be the package manager that manages this file rather than allowing the user to edit it manually.

**Comment by johnfanv2** [comment]
@st0nie Please discuss your feature in a isssue with @MrDuartePT .

**Pull request merged**